source/docs/software/commandbased/binding-commands-to-triggers.rst

```diff
@@ -129,8 +131,8 @@ cancelWhenActive/cancelWhenPressed
 
 This binding cancels a command when a trigger changes from inactive to
 active (or, accordingly, when a button is initially pressed). the
-command is canceled on the iteration when the state changes, and will
-not be canceled again unless the trigger becomes inactive and then
+command is cancelled on the iteration when the state changes, and will
```

Comment:
Very British of us.

Reply:
Oops, my Notepad++ was set to British for some reason and I didn't think twice about it.